### DISCUSSION TITLE
Bienvenue à toi, cher(e) volontaire

### DISCUSSION THREAD

**Discussion by dofbi** [Code-for-Senegal/joinDjolof#1] [Announcements]
## 👋 Bienvenue!
  Nous utilisons Discussions comme lieu de connexion avec d'autres volontaires de notre communauté **Djolof**. Nous espérons que vous :
  * Posez les questions que vous vous posez.
  * Partager des idées.
  * S'engager avec d'autres volontaires de la communauté.
  * Accueillez les autres et faites preuve d'ouverture d'esprit. N'oubliez pas qu'il s'agit d'une communauté que nous ensemble. 💪.

Pour commencer, commentez ci-dessous avec une présentation de vous-même 🙏🏾 .

**Comment by MedouneSGB** [comment]
Bonjour, je m'appelle Médoune Siby Georges Baldé 😁
Je suis Développeur Logiciel et Consultant ERP.
Content de pouvoir participer. 💯✨

**Reply by dofbi** [reply]
Salaam grand @MedouneSGB 
Bienvenue 😉

**Reply by MedouneSGB** [reply]
Merci Master !

**Comment by EmmanuelDiokh** [comment]
Bonjour Emmanuel Diokh
Je suis juriste en droit numérique et passionnée par l'einovation technologique.

**Reply by dofbi** [reply]
Hello @EmmanuelDiokh 
Bienvenue 😉

**Comment by ByNatO** [comment]
Bonjour,

Omar Badiane, ingénieur en informatique et Télécom, AI enthusiast.

**Reply by dofbi** [reply]
Hello @ByNatO 
Bienvenue 😉

**Comment by ByNatO** [comment]
Exemple d'une instance: https://scholar.social/about#rules

**Reply by dofbi** [reply]
Merci pour le partage 👍🏾
@EmmanuelDiokh est un avocat et légal Tech qui pourra travailler avec toi sur les règles 🇸🇳

**Reply by EmmanuelDiokh** [reply]
Haha grand avocat yi degniy mer dé 
En fait je suis juriste spécialisé en droit du numérique. Depuis 10 ans je m'interesse qux question de droit liées aux technologies numériques.

**Reply by dofbi** [reply]
Merci @EmmanuelDiokh pour la précision (Sunu Avoca 😄 )
J'ai crée un issue ici #2

**Comment by AlaajiAbdulaay** [comment]
Bonjour,
Alaaji Abdulaay
Blogueur

**Reply by ByNatO** [reply]
Bonjour Alaaji. Bienvenue!

**Comment by SadikhNdiaye** [comment]
Aboubacar sadikh ndiaye
Consultant en #transfoNum

**Comment by ByNatO** [comment]
Bonjour team: link sur la création d'instance Mastodon  https://docs.joinmastodon.org/user/run-your-own/

**Reply by dofbi** [reply]
Merci pour le partage @ByNatO ,
n'hésite pas à rejoindre la discussion #7 si tu as envie de contribuer dans l'équipe serveur 🙏🏾
